### ISSUE TITLE
Deleting categories?

### ISSUE THREAD

**Issue by haraldkubota** [sirceljm/aws-lambda-blog#16]
Hi,

Is it possible that deleting categories is not working? It looks ok on the front-end, but all deleted categories come back when I do a web browser reload.
Renaming and adding works as expected.

Harald

**Comment by pbran** [comment]
I wondered if this was a browser issue but I have tried with Firefox and Chrome on Mac OS and both are the same. The remove_category.js function logs a No Token error in CloudWatch and DynamoDB doesn't remove the entry. I have tracked it down to a missing line from the API Gateway for the Body Mapping template for remove-category .
"token_name": "${stageVariables.token_name}",

 I have noticed that adding a new category does add it successfully but since there is no feedback in the browser when you press the Add button you are tempted to press it again and add it more than once.
